.github/workflows/publish.yml

```
@@ -15,6 +22,7 @@ jobs:
         pip3 install -r requirements.txt
     - name: Sphinx build
       run: |
+        git clone --depth 1 https://github.com/key4hep/k4marlinwrapper
```

```suggestion
        curl --fail --silent https://raw.githubusercontent.com/key4hep/k4MarlinWrapper/master/doc/starterkit/k4MarlinWrapperCLIC/Readme.md -o examples/clic.md
```
if the other files are not needed? and curl is available?

---

Hmm, that obscures where the file is coming from, and we might want to add other pages like https://github.com/key4hep/k4MarlinWrapper/blob/master/doc/edmConverters.md

---

```suggestion
        curl --fail --silent https://raw.githubusercontent.com/key4hep/k4MarlinWrapper/master/doc/starterkit/k4MarlinWrapperCLIC/Readme.md -o k4marlinwrapper/doc/starterkit/k4MarlinWrapperCLIC/Readme.md
```
I propose to do this, and tackle other pages in a future PR.